resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/repository/BackingStore.java

```diff
@@ -44,6 +44,58 @@
 )
 public interface BackingStore {
 
```

Comment:
It looks to me as if a lot of the changes here are just whitespace changes - are you using the gaffer xml template ???

Reply:
This seems mostly unavoidable as this was all in the middle of our transition to using checkstyle.
It will be better separated into unittests/refactoring/checkstyle  next time